Makefile

```diff
@@ -39,6 +40,10 @@ docker:
 docker-run: docker ## Build and run the application in a local docker container
 	@docker run -p ${DEFAULT_APP_PORT}:${DEFAULT_APP_PORT} -v ${PWD}${DEFAULT_CONFIG_FILE}:${DEFAULT_CONFIG_FILE} ${ENV_VARIABLES} $(CMD_NAME):latest --config-file=${DEFAULT_CONFIG_FILE}
 
+.PHONY: docker-push-local
+docker-push-local: ## Build and push the image to local docker registry
+	@docker build -t $(LOCAL_REGISTRY)/$(CMD_NAME):latest .
+	@docker push $(LOCAL_REGISTRY)/$(CMD_NAME):latest
```

Comment:
Not sure if this goes against our open-source public repo design guidelines. I would like this to easily push to our minikube environment and test. But I'm okay with taking it out as long as there's a similarly easy way to test locally against our Kanopy configuration.

Reply:
This seems fine to me.